components/TextFields/examples/TextFieldStatesViewController.m

```diff
@@ -81,29 +81,38 @@ - (void)viewDidLoad {
                                attribute:NSLayoutAttributeLeadingMargin
                               multiplier:1
                                 constant:0]
-  .active = YES;
+      .active = YES;
   [NSLayoutConstraint constraintWithItem:self.enabled
                                attribute:NSLayoutAttributeTrailing
                                relatedBy:NSLayoutRelationEqual
                                   toItem:self.view
                                attribute:NSLayoutAttributeTrailingMargin
                               multiplier:1
                                 constant:0]
-  .active = YES;
+      .active = YES;
   [NSLayoutConstraint constraintWithItem:self.enabled
                                attribute:NSLayoutAttributeTrailing
                                relatedBy:NSLayoutRelationEqual
                                   toItem:self.scrollView
                                attribute:NSLayoutAttributeTrailingMargin
                               multiplier:1
                                 constant:0]
-  .active = YES;
-
-  NSDictionary *views = @{@"enabled": self.enabled,
-                          @"disabled": self.disabled,
-                          @"errored": self.errored,
-                          @"erroredDisabled": self.erroredDisabled};
-  [NSLayoutConstraint activateConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:[enabled]-[disabled]-[errored]-[erroredDisabled]" options:NSLayoutFormatAlignAllLeading | NSLayoutFormatAlignAllTrailing metrics:nil views:views]];
+      .active = YES;
+
+  NSDictionary *views = @{
```

Comment:
This looks like a bug in the formatter? Perhaps it thinks you're opening a new scope?